TECHMANIA/Assets/Scripts/Serializable/Options.cs

```diff
@@ -197,6 +197,9 @@ public void ApplyGraphicSettings()
         //
         // Resolution is not supported at all on iOS.
         Screen.SetResolution(width, height, fullScreenMode, refreshRate);
+#elif !UNITY_ANDROID
+		Screen.SetResolution(width, height, fullScreenMode, refreshRate);
+		Application.targetFrameRate = 360;
```

Comment:
360 is quite an arbitrary number and probably won't work for everyone.

Look at the #if block below this line - it tells iOS to tie frame rate to refresh rate, and ignore VSync. Try to apply it to UNITY_ANDROID too, and maybe it will just work for you? Please let us know.

Reply:
Unfortunately, the proposed method has not solved it. 360 is an arbitrary number and is rendered only by the maximum scanning rate, not by the maximum frame of 360.

Reply:
That's exactly what `Application.targetFrameRate = Screen.currentResolution.refreshRate;` should achieve. When applying it to Android what frame rate do you get? And what is your device's refresh rate?

Reply:
All of my devices supported 120 Hz and were still fixed at 30 frames when executed in the above way.

Reply:
Just to be absolutely sure, this is my proposal in code (added a line to set vSyncCount to 0 just in case):
```
    public void ApplyGraphicSettings()
    {
#if !UNITY_ANDROID && !UNITY_IOS
        // Setting resolution on Android causes the graphics to
        // be stretched in the wrong direction.
        //
        // Resolution is not supported at all on iOS.
        Screen.SetResolution(width, height, fullScreenMode, refreshRate);
#endif

#if UNITY_IOS || UNITY_ANDROID
        // iOS ignores VSync, and caps the FPS at 30 by default.
        Application.targetFrameRate =   
            Screen.currentResolution.refreshRate;
        QualitySettings.vSyncCount = 0;
#else
        QualitySettings.vSyncCount = vSync ? 1 : 0;
#endif
    }
```
Please give it a try, and tell us whether you are still locked at 30 FPS.

Reply:
I will try it asap.

Reply:
So cool. It's working well.